db/models/article.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../connection");
+
+exports.selectArticle = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
```

Comment:
Great to see you are making use of a parameterized query to protect against SQL injection 👍